modules/signing_worker/templates/scriptworker_config.erb

```diff
@@ -0,0 +1,30 @@
+work_dir: /tmp/work
+artifact_dir: /tmp/artifacts
+taskcluster_scope_prefix: "project:releng:signing:"
+verbose: true
+local_notarization_accounts: ["account1"]
+concurrency_limit: 2
+7z: /usr/ocal/bin/7z
+mac_config:
+    dep:
```

Comment:
We'll also need to populate `nightly` and `release` blocks in `mac_config`, which look like `dep` but different.